app/javascript/courses/review/components/CoursesReview__Root.re

```diff
@@ -321,11 +377,17 @@ let make =
 
   let url = ReasonReactRouter.useUrl();
 
-  let filteredPendingSubmissions =
-    state.pendingSubmissions
-    |> filterSubmissions(state.selectedLevel, state.selectedCoach)
-    |> IndexSubmission.sort;
-
+  let filteredPendingSubmissions = {
+    let filteredSubmissions =
+      state.pendingSubmissions
+      |> filterSubmissions(state.selectedLevel, state.selectedCoach);
+    let sortedSubmissions =
+      switch (state.sortDirection) {
+      | `Up => filteredSubmissions |> IndexSubmission.sortUp
+      | `Down => filteredSubmissions |> IndexSubmission.sortDown
+      };
+    sortedSubmissions;
```

Comment:
Since the sorting functions are in the module, why not just have a single function there that accepts the direction as well?

```suggestion
    filteredSubmissions |> IndexSubmission.sortArray(state.sortDirection)
```